include/ofi_util.h

```diff
@@ -209,6 +209,8 @@ struct util_domain {
 	uint64_t		info_domain_caps;
 	uint64_t		info_domain_mode;
 	int			mr_mode;
+	uint64_t		tx_msg_order;
+	uint64_t		rx_msg_order;
```

Comment:
This ordering can change per endpoint.  We'll need to figure out how to deal with this, or force all endpoints to use the same ordering.

It's highly likely all endpoints will have the same ordering.  But that's not guaranteed, especially since this is updating the util code.